tests/php/test-class-amp-gallery-block-sanitizer.php

```diff
@@ -92,6 +93,123 @@ public function get_data() {
 				'<figure class="wp-block-gallery" data-amp-lightbox="true" data-amp-carousel="true"><ul><li class="blocks-gallery-item"><figure><a href="http://example.com"><amp-img src="http://example.com/img.png" width="1234" height="567"></amp-img></a></figure></li></ul><figcaption>This is the gallery caption</figcaption></figure>',
 				'<figure class="wp-block-gallery" data-amp-lightbox="true" data-amp-carousel="true"><amp-carousel width="1234" height="567" type="slides" layout="responsive" lightbox=""><figure class="slide"><amp-img src="http://example.com/img.png" width="1234" height="567" layout="fill" object-fit="cover"></amp-img></figure></amp-carousel><figcaption>This is the gallery caption</figcaption></figure>',
 			],
+
+			'carousel_from_wp59_gallery_block_markup' => [
+				'
+				<figure class="wp-block-gallery has-nested-images columns-default is-cropped" data-amp-carousel="true">
+					<amp-carousel width="2560" height="1920" type="slides" layout="responsive">
```

Comment:
This and other new test cases have an `amp-carousel` in the `$source` markup. Is that expected? Shouldn't we test the core Gallery block markup, i.e. `figure.wp-block-gallery > figure` instead of `figure.wp-block-gallery > amp-carousel > figure`?

Reply:
Oops!

Reply:
Fixed in 920eaa1ff.

Reply:
Looks good!